OpenRA.Game/Network/Session.cs

```diff
@@ -218,6 +218,9 @@ public class Global
 			public bool Dedicated;
 			public bool GameSavesEnabled;
 
+			// 120ms network frame interval for 40ms local tick
+			public int NetFrameInterval = 3;
```

Comment:
Note that any value below 3 seems to cause slowdowns in singleplayer, because it uses hardcoded orderlatency 1 (Server.cs line 65) which (as far as I understand) causes stalls while waiting for the order to come back from the server (there is zero buffering).

Reply:
Yes, this will be addressed by removing the hardcoded behaviour and allowing the dynamic latency system to handle it.